api/aperture/common/peers/v1/peers.proto

```diff
@@ -2,6 +2,31 @@ syntax = "proto3";
 
 package aperture.common.peers.v1;
 
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+//
+// gRPC service
+//
+
+// PeerDiscoveryService is used to query Peer discovery.
+service PeerDiscoveryService {
+  rpc GetPeers(google.protobuf.Empty) returns (Peers) {
+    option (google.api.http) = {
+      get: "/v1/peers/peers"
```

Comment:
can the same API be used to get all peers and specific peers?

e.g. calling `/v1/peers/` gets all peers.
and calling `/v1/peers/{address}` gets a specific peer.

Please see how we do it for the status registry.

Reply:
Got it - using the nested structure like as `GroupStatus` which holds status information and the pointer to the next node, I can have the same API, `GetPeers` to get both peers and specific peer with the given address.